README-dev.md

```diff
@@ -0,0 +1,49 @@
+# How to develop SnarkyJS
+
+This README includes information that is helpful for SnarkyJS core contributors.
+
+## Run examples in a node
+
+```sh
+npm install
+npm run build
+
+./run src/examples/api_exploration.ts
+```
+
+## Build and run the web version
+
+```sh
+npm install
+npm run build:web
+npm run serve:web
+```
+
+To see the user interface in a web browser, go to `http://localhost:8000/`
```

Comment:
do we need to provide advice on barriers, like firewall or browser rule blocking?

Reply:
There's no user interface, just a test that you can see running in the browser dev tools, so should change this back / to something else :D